.circleci/config.yml

```diff
@@ -60,13 +72,14 @@ jobs:
           command: npm run semantic-release
 
 workflows:
-  build:
+  lint-build-release:
     jobs:
       - lint-and-test:
           matrix:
             parameters:
               node-version: ["14.15.0", "16.13.0", "18.12.0"]
       - lint-commits
+      - build
```

Comment:
This makes it so on this workflow it gets build (project and docs) twice right? Here and in the prepublish

Reply:
Yes. My idea was to run the build separately to fail early. Otherwise, we wait for lining and tests to pass before we try whether it actually builds.
Also, the `prepublish` is not called when semantic-release is run in dry-mode. -> We currently do not build the package in out PRs